cmd/main.go

```diff
@@ -94,6 +94,7 @@ const (
 	deployment_type_ocp    = "ocp"
 	storage_filesystem     = "filesystem"
 	storage_s3             = "s3"
+	hostFsMountDir         = "/host"
```

Comment:
I think we should consistently name this as `hostFSMountDir` right now the FS is capitalized in the instruction config, but not here or in the reconciler. I think it should always be `FS`

Reply:
Gotcha, modified it!